README.md

```diff
@@ -88,6 +89,53 @@ Now, **goManageDocker 😏!!**
    
    ![search](vhs/gifs/search.gif)
 
+## Keybinds
+
+### Navigation
+| Operation | Key                                                                 |
+|-----------|---------------------------------------------------------------------|
+| Enter     | <kbd>Enter</kbd>                                                    |
+| Back      | <kbd>Esc</kbd>                                                      |
+| Quit      | <kbd>Ctrl</kbd> + <kbd>c</kbd> / <kbd>q</kbd>                       |
+| Next Tab  | <kbd>→</kbd> / <kbd>l</kbd> / <kbd>Tab</kbd>                        |
+| Prev Tab  | <kbd>←</kbd> / <kbd>h</kbd> / <kbd>Shift</kbd> + <kbd>Tab</kbd>     |
+| Next Item | <kbd>↓</kbd> / <kbd>j</kbd>                                         |
+| Prev Item | <kbd>↑</kbd> / <kbd>k</kbd>                                         |
+| Next Page | <kbd>[</kbd>                                                        |
+| Prev Page | <kbd>]</kbd>                                                        |
+
+### Image
+| Operation         | Key                                                           |
+|-------------------|---------------------------------------------------------------|
+| Run               | <kbd>r</kbd>                                                  |
+| Rename            | <kbd>R</kbd>                                                  |
+| Scout             | <kbd>s</kbd>                                                  |
+| Prune             | <kbd>p</kbd>                                                  |
+| Delete            | <kbd>d</kbd>                                                  |
+| Delete Force      | <kbd>D</kbd>                                                  |
+| Copy Id           | <kbd>c</kbd>                                                  |
+| Run and Execute   | <kbd>x</kbd>                                                  |
+
+### Container
+| Operation         | Key                                                           |
+|-------------------|---------------------------------------------------------------|
+| Toggle List All   | <kbd>a</kbd>                                                  |
+| Toggle Start/Stop | <kbd>s</kbd>                                                  |
+| Toggle Pause      | <kbd>t</kbd>                                                  |
+| Restart           | <kbd>r</kbd>                                                  |
+| Delete            | <kbd>d</kbd>                                                  |
+| Delete Force      | <kbd>D</kbd>                                                  |
+| Execute           | <kbd>x</kbd>                                                  |
```

Comment:
```suggestion
| Exec          | <kbd>x</kbd>                                                  |
```

Reply:
Should "Run and Execute" also be "Run and Exec" or stay as is

Reply:
Yes! Please change that as well.